feature/sessions/src/commonTest/kotlin/io/github/droidkaigi/confsched/sessions/TimetableItemDetailScreenTest.kt

```diff
@@ -58,8 +69,13 @@ class TimetableItemDetailScreenTest {
                     }
                 }
 
-                // TODO https://github.com/DroidKaigi/conference-app-2025/issues/218
-                // TODO Test that both the transition methods to the video and slide assets are hidden.
+                itShould("not show the slides button and video button") {
+                    captureScreenWithChecks(
+                        checks = {
+                            checkBothOfSlidesButtonAndVideoButtonNotDisplayed()
+                        },
+                    )
+                }
```

Comment:
This itShould invocation is also an unintended usage, which causes the screenshots to not be as intended.
How about asserting this condition inside the scroll behavior test case instead?

<img width="651" height="536" alt="image" src="https://github.com/user-attachments/assets/b07dd6c8-ffd1-46e5-89e2-e9e93a98de1a" />